karton/emit.py

```diff
@@ -143,8 +143,10 @@ def _emit_system_packages(self):
 
         if self._props.sudo != DefinitionProperties.SUDO_NO:
             self._emit_install('sudo')
-
-        self._emit_install('python')
+            
+        # Fix: https://github.com/karton/karton/issues/15
+        # note: can be python2.7 or 3
```

Comment:
I think we should just have Python 3 at this point. It's 2022 after all :)
No need for the comment as, at this point, we should just accept that Python 3 is the default.